pinot-broker/src/main/java/org/apache/pinot/broker/broker/BrokerAdminApiApplication.java

```diff
@@ -105,6 +105,18 @@ protected void configure() {
         bind(accessFactory).to(AccessControlFactory.class);
       }
     });
+    boolean enableBoundedThreadPool = brokerConf
+            .getProperty(CommonConstants.Broker.CONFIG_OF_ENABLE_BOUNDED_THREAD_POOL,
+                    CommonConstants.Broker.DEFAULT_ENABLE_BOUNDED_THREAD_POOL);
+    if (enableBoundedThreadPool) {
```

Comment:
nit: move initialization to a private method

```
register(buildBrokerManagedAsyncExecutorProvider(...))
```

Reply:
Thanks, addressed in the second commit.